easybuild/easyconfigs/p/Paraver/Paraver-4.11.4-foss-2023a.eb

```
@@ -0,0 +1,21 @@
+name = 'Paraver'
+version = '4.11.4'
+
+homepage = 'https://tools.bsc.es/paraver'
+description = """A very powerful performance visualization and analysis tool based on
+ traces that can be used to analyse any information that is expressed on its input trace format.
+ Traces for parallel MPI, OpenMP and other programs can be genereated with Extrae."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Paraver probably belongs in the GCC level, not foss
I can't find any trace of it needing neither MPI nor BLAS/FFTW

---

Hmm, will try to ask Paraver developers

---

@boegel asked @julianmorillo, and he confirmed that Paraver does not depend on MPI/BLAS/LAPACK/FFTW.

```suggestion
toolchain = {'name': 'GCC', 'version': '12.3.0'}
```